users.html

```diff
@@ -149,6 +149,9 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // AIRCAS
   L.marker([40.069025729587736, 116.27675096818587], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Aerospace Information Research Institute with the Chinese Academy of Sciences (AIRCAS)</strong><br>Beijing, China<ul><li><a href='http://english.aircas.ac.cn/' target='_new'>Website</a></li></ul>");
 
+  // Altai State University
+  L.marker([53.348202295417536, 83.77583607976237], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Altai State University</strong><br>Barnaul, Russia<ul><li><a href='https://www.asu.ru/en/' target='_new'>Website</a></li></ul>");
```

Comment:
@msulprizio: The URL is correct but the site times out.  Maybe due to network issues.